services/Directory.Build.props

```diff
@@ -0,0 +1,15 @@
+<Project>
+    <PropertyGroup>
+        <Deterministic>true</Deterministic>
+        <Features>strict</Features>
```

Comment:
note: we use deterministic builds here because that means the compiler will produce the exact same binary output (and be sped up by a factor of 70%) which is good in situations like testing, where the actual project doesn't change, but the tests do (or really any consumer of a library) - and strict features to remove some intentional bugs in the compiler (microsoft added intentional bugs, replicating old bugs from the old compiler for backwards compat, cheers MS!)